javascript/boilerplate-text/boilerplate-fr.js

```diff
@@ -55,6 +55,7 @@ s.examplesInAnotherScript = "Ce document contient des exemples dans une autre la
 s.worldMap = "Carte du monde";
 s.searchI18nSite = "Recherche sur le site I18N"; 
 s.translationDisclaimer = 'Ce document est une traduction fournie bénévolement. En cas de divergences ou d\'erreurs, la <a href="'+f.filename+'.en">dernière version originale en anglais</a> fait autorité. Comme indiqué ci-dessous, les <a href="#copyright">droits d\'auteur</a> reviennent au W3C.'
+s.translator = "Traducteur :";
```

Comment:
This line is no longer in the latest version of this file. (s.translatedBy is used instead)